api/types/payloads.go

```diff
@@ -52,3 +54,25 @@ type SockVariantDTO struct {
 	Price    float64 `json:"price" validate:"required,gt=0,lt=101"`
 	Quantity int     `json:"quantity" validate:"required,gte=0"`
 }
+
+type GetOrdersRequest struct {
```

Comment:
You can remove this. It's not used and GET request don't have payloads just query params.